modules/tide_landing_page/tide_landing_page.module

```diff
@@ -605,3 +608,19 @@ function tide_landing_page_node_presave(NodeInterface $node) {
     }
   }
 }
+
+/**
+ * Implements hook_entity_base_field_info_alter().
+ */
+function tide_landing_page_entity_base_field_info_alter(&$fields, EntityTypeInterface $entity_type) {
+  if ($entity_type->id() == 'node') {
```

Comment:
So this is in the tide_landing_page module, but the behavior applies to all node bundles?

Reply:
thank you @nicksantamaria for the CR. yes. this is not appropriate, the whole change should be moved to the `tide_site` submodule, since this PR also needs functionality from `tide_site`.

Reply:
I agree with Nick here. I know we use "landing page" as a bit of a catch all, but we should be recognising this is common/shared functionality.